src/components/PcpFeed/PcpCard/index.js

```diff
@@ -0,0 +1,114 @@
+import * as moment from "moment-timezone";
+import PropTypes from "prop-types";
+import { makeStyles } from "tss-react/mui";
+
+const useStyles = makeStyles()((theme) => ({
+	container: {
+		color: theme.palette.text.primary,
+		background: "white",
+		padding: "0.6rem 0 0.6rem 0",
+		display: "flex",
+		gap: "0.625rem",
+		flexDirection: "row",
+		width: "100%",
+		borderBottom: "1px solid #585858",
+	},
+	card: {
+		width: "100%",
+	},
+	header: {
+		fontWeight: 700,
+	},
+	content: {
+		display: "flex",
+		flexDirection: "row",
+		justifyContent: "space-between",
+		flexWrap: "wrap",
+	},
+	status: {
+		display: "inline-flex",
+		marginLeft: "auto",
+		"& div": {
+			fontWeight: 700,
+			padding: "0.4rem 0.875rem 0.4rem 0.875rem",
+			borderRadius: "1rem",
+			fontSize: "0.875rem",
+			lineHeight: "1rem",
+			color: "white",
+			display: "flex",
+			justifyContent: "center",
+			width: "5.624rem",
+		},
+	},
+	closed: {
+		background: "#EF0A0A",
+	},
+	open: {
+		background: "#31A35F",
+	},
+	upcoming: {
+		background: "#FCBA19",
+	},
+}));
+
+const PcpCard = ({ dateCompleted, dateStarted, phaseName, projectName }) => {
+	const { classes } = useStyles();
+
+	const formattedDate = (date) => {
+		return new Intl.DateTimeFormat("en-US", {
+			year: "numeric",
+			month: "long",
+			day: "numeric",
+		}).format(new Date(date));
+	};
+
+	const status = (startDate, endDate) => {
+		if (startDate && endDate) {
+			const now = new Date();
+			const dateStarted = moment(startDate);
+			const dateCompleted = moment(endDate);
+			if (moment(now).isBetween(dateStarted, dateCompleted)) {
```

Comment:
Please use `date-fns` equivalents or add a js Date functions to the date service for these comparisons.

Reply:
Addressed in c53b5c2